src/Backend/Core/Engine/Meta.php

```diff
@@ -477,6 +477,11 @@ public function save($update = false)
             $custom = null;
         }
 
+        //serialize data for save
+        if (!empty($this->data['data'])) {
+            $this->data['data'] = serialize($this->data['data']);
```

Comment:
that way your code will come into master when it is merged and won't come in other PR you make